tst/EnergyPlus/unit/Furnaces.unit.cc

```diff
@@ -87,7 +87,7 @@ namespace EnergyPlus {
 TEST_F(EnergyPlusFixture, SetVSHPAirFlowTest_VSFurnaceFlowTest)
 {
 
-    state->init_state(*state);
+    state->init_state(*state); // get FluidProperties
```

Comment:
hmm, why did I have to leave this here when FluidProperties is called in the fixture? Maybe this unit test needed something else now in init_state?

Reply:
Oh, that's right. If the unit test does not have an idf snippet then process_idf will not be called and default fluids will not be set up yet. So this is correct.